MapboxNavigation/Resources/Assets.xcassets/feedback/confusing_audio.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "filename" : "confusing_audio.pdf",
+      "idiom" : "universal",
+      "language-direction" : "left-to-right"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  },
+  "properties" : {
+    "preserves-vector-representation" : true
+  }
```

Comment:
In the asset catalog, change “Render As” to from “Original Image” to “Template Image” so that the icons will adopt the application’s tint color automatically.

Reply:
Hmm, If i do this the icons seem to disappear. Example:

![Simulator Screen Shot - iPhone 8 Plus - 2020-06-24 at 22 20 39](https://user-images.githubusercontent.com/6844889/85645918-fe729e80-b668-11ea-857f-8b1dd7e65794.png)


(I've left `Route Preview` as `default` in order to show the difference)

Reply:
The tint color has always been set to clear, bizarrely enough:

https://github.com/mapbox/mapbox-navigation-ios/blob/fa19891a24b90596169046468947a79c7cd76b75/MapboxNavigation/FeedbackViewController.swift#L286

I think we can simply remove that line; the view controller will inherit the application’s tint color.

Reply:
Hmm, i think these images aren't imported correctly as template images. I see this if I remove that line and make the change to "Render as: template":

![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-06-29 at 10 50 56](https://user-images.githubusercontent.com/6844889/86020991-991b0680-b9f6-11ea-89ce-bad43599be41.png)

Reply:
Added a note in the description of https://github.com/mapbox/mapbox-navigation-ios/pull/2429 to handle this issue.

Reply:
The problem is that the glyphs within the circles are white instead of transparent. We’ll need to make them transparent so that the template image will show up correctly.

Reply:
mapbox/navigation-ui-resources#15

Reply:
#2433 tracks tail work around tint colors.